### PR TITLE
Added Cookiebot SDK typings

### PR DESCRIPTION
**Pull request by LiahMartens**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/cookiebot-sdk/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+// Type definitions for Cookiebot SDK
```
Comment (gabritto) [inline types/cookiebot-sdk/index.d.ts:1]: This line should read `// Type definitions for non-npm package Cookiebot SDK` (and possibly a version number to which these typings refer).

File: types/cookiebot-sdk/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "rules": {
+        "dt-header": "off"
```
Comment (gabritto) [inline types/cookiebot-sdk/.eslintrc.json:3]: I don't think you'll need this file once the header is fixed. Please remove it.

File: types/cookiebot-sdk/tslint.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+    "npm-naming": false
```
Comment (gabritto) [inline types/cookiebot-sdk/tslint.json:4]: Please fix the header in `index.d.ts` and remove this `"rules"` property.

**Review by gabritto — CHANGES_REQUESTED** [review]
Looks good except for the issue with the header in `index.d.ts`. Please fix that and I'll approve this PR. Thanks.

**Review by andrewbranch — APPROVED** [review]

**Comment by LiamMartens** [comment]
Ready to merge

**Pull request merged**